src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <_CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
Please use `CodeCoverageAssemblies` instead of `_CodeCoverageAssemblies` (https://github.com/dotnet/corefx/blob/7642f1f980fb0e8fa45f67769e0755a97df7b4d5/Tools-Override/CodeCoverage.targets#L44). The underscore properties is our convention for private msbuild properties which shouldn't be used outside of the scope of where they are defined and used. Unfortunately it isn't enforceable by msbuild.